src/esm_parser/yaml_to_dict.py

```diff
@@ -199,9 +201,11 @@ def yaml_file_to_dict(filepath):
                     yaml_load["computer"]["runtime_environment_changes"][
                         "add_export_vars"
                     ] = add_export_vars
-                # Check for empty components/models
-                # check_for_empty_components(yaml_load, filepath + extension)
-                return yaml_load
+
+            yaml_load = DictWithProvenance(yaml_load, {"filepath": filepath})
```

Comment:
This looks suspiciously "too easy"

Reply:
This is just a test, in Nadine's work is a bit more complicated, but in conceptually is that easy ;)